packages/augur-sdk/src/state/getter/Markets.ts

```diff
@@ -468,15 +469,20 @@ export class Markets {
       },
       isbids: boolean = false
     ): OrderBook[] => {
-      const orders = Object.values(unsortedOrders).sort((a, b) =>
-        isbids
-          ? new BigNumber(b.price).minus(a.price).toNumber()
-          : new BigNumber(a.price).minus(b.price).toNumber()
-      );
+      const orders = Object.values(unsortedOrders);
       const buckets = _.groupBy<Order>(orders, order => order.price);
+      const keysSorted: string[] = isbids
```

Comment:
I think this should by DRYd up a bit, it took me a couple reads

Reply:
If you have some suggestions. It's confusing by nature of building the orderbook